java/src/apps/gui3/FirstTimeStartUpWizard.java

```diff
@@ -364,7 +375,7 @@ JLabel formatText(String text) {
 
     JComboBox<String> localeBox;
     HashMap<String, Locale> locale;
-    String[] localeNames;
+    final String[] localeNames;
```

Comment:
Where is this variable used? You initialize it, but is it used anywhere?
I'm on mobile phone now so I have difficult to read the code.
Static analysis wouldn't catch unused variables that are protected or package private.

Reply:
localNames[] is used to populate the locale JComboBox.
On about 10% of test runs, the Thread which writes to this Array cannot find the Array, hence initialising in construction.